spec/models/user_spec.rb

```diff
@@ -87,13 +87,62 @@
   end
 
   describe 'methods' do
+    describe '#attended_event?' do
+      context 'user has attended to the event' do
+        before do
+          registration = create(:registration, user: user, conference: conference)
```

Comment:
do this in a let, then it is created when needed, can be reused and then you have all the object created for your tests together and at the beginning. 

And as you are using it for both `#attended_event?` and `#registered_to_event?` you can move it up (just after `describe 'methods' do`) so you only need to define it once. :slightly_smiling_face: